dali/kernels/signal/dct/dct_gpu_test.cc

```diff
@@ -27,13 +28,24 @@ namespace dct {
 namespace test {
 
 class Dct1DGpuTest : public ::testing::TestWithParam<
-  std::tuple<int, std::pair<int, std::vector<int>>>> {
+  std::tuple<int, float, std::pair<int, std::vector<int>>>> {
  public:
   Dct1DGpuTest()
       : batch_size_(std::get<0>(GetParam()))
-      , dims_(std::get<1>(GetParam()).first)
-      , axes_(std::get<1>(GetParam()).second)
+      , lifter_(std::get<1>(GetParam()))
+      , dims_(std::get<2>(GetParam()).first)
+      , axes_(std::get<2>(GetParam()).second)
       , in_shape_(batch_size_, dims_) {
+        if (lifter_) {
+          FillLifter();
+          const int max_ndct = 40;
```

Comment:
Where this 40 comes from?
I see it repeated in L65, maybe extract it to a common place.

Reply:
done